src/open_inwoner/openzaak/catalog.py

```diff
@@ -202,6 +193,34 @@ def fetch_single_case_type(case_type_url: str) -> Optional[ZaakType]:
     return case_type
 
 
+@cache_result(
```

Comment:
This function exists twice in the file, maybe after rebasing.

Reply:
Well spotted!

We really need to clean these at some point because what started as a few handy helper is growing out of control. I made a ticket 1096 on Taiga for this.